lib/generators/spectr/USAGE

```diff
@@ -0,0 +1,8 @@
+Description:
+    Create spectr.yml in Rail.root/config folder which will be user by this gem to add database details
+
+Example:
+    rails generate spectr Thing
```

Comment:
Why a Named generator is needed.

Reply:
In future if we need to pass arguments from command line then we will need it